lib/shopify_app/version.rb

```diff
@@ -1,4 +1,4 @@
 # frozen_string_literal: true
 module ShopifyApp
-  VERSION = '15.0.1'
+  VERSION = '15.1.0'
```

Comment:
Should we consider making this a major bump? Even though it doesn't break anything it will affect the styling of existing apps, which may become inconsitent.

Reply:
Yeah I can definitely update it, I was on the fence between the two.
I referred to previous styling updates (eg. bump to version 8.2.0) where it was just a minor bump but this is a bit of a larger change so major makes sense to me.